modules/ROOT/pages/index.adoc

```diff
@@ -5,25 +5,25 @@ endif::[]
 :keywords: hl7, healthcare, toolkit, hapi, mllp, er7
 :license-info: Enterprise, CloudHub
 
-Mule Healthcare Toolkit is a collection of features that facilitates integration with healthcare systems by providing the tools needed to easily create, read, and transform HL7 messages within Mule.
+Healthcare Toolkit is a collection of features that facilitates integration with healthcare systems by providing the tools needed to easily create, read, and transform HL7 messages within Mule.
 
-This document assumes a working knowledge of Mule Runtime and Anypoint Studio.
+This document assumes a working knowledge of Mule and Studio.
```

Comment:
First mention of Studio should be as Anypoint Studio to preserve branding. Also, please add Anypoint Platform to this list.